docs/api/arrayfunctions.rst

```diff
@@ -14,3 +14,19 @@ Array functions
 .. autofunction:: apytypes.swapaxes
 
 .. autofunction:: apytypes.expand_dims
+
+.. automethod:: apytypes.zeros
```

Comment:
`autofunction`? (As above. They may be the same, but for consistency and these are functions.)